hid_devices.py

```diff
@@ -136,9 +186,8 @@ def move_mouse(self, xy: bytes) -> None:
         self.device_registry.bluetooth_devices.send_message(b'\xa1\x03\x00\x00\x00\x00' + xy, True, False)
 
     async def send_message(self, msg: bytes) -> None:
-        tm = self.filter.filter_message_from_host(msg)
```

Comment:
Are we good removing this? I've not seen any use of this and struggle to think of any uses.